bindings/netstandard/ElectionGuard/ElectionGuard.Encryption.Tests/TestPrecompute.cs

```diff
@@ -1,119 +1,81 @@
-﻿using System.Threading;
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
 using NUnit.Framework;
 
 namespace ElectionGuard.Encryption.Tests
 {
+    [NonParallelizable]
     [TestFixture]
-    class TestPrecompute
+    public class TestPrecompute

```

Comment:
this test was made to validate the state machine so i made it non-parallelizable and the tests execute in order now.